internal/deviations/deviations.go

```diff
@@ -72,20 +74,60 @@ import (
 	"github.com/openconfig/ondatra"
 )
 
+func matchRegex(regex string, deviceInfo string) bool {
+	match, err := regexp.MatchString(regex, deviceInfo)
+	if err != nil {
+		log.Errorf("Error with regex match %v", err)
+	} else if match {
+		return true
+	}
+	return false
+}
+
 // lookupDutDeviations returns the deviations for the specified dut.
```

Comment:
this comment is not styled correctly "Dut" should be "DUT", but it doesn't seem to add anything beyond what the method name already communicates, so I'd just drop it entirely.

Reply:
done